examples/jsm/loaders/GLTFLoader.d.ts

```diff
@@ -29,3 +31,19 @@ export class GLTFLoader extends Loader {
 	parse( data: ArrayBuffer | string, path: string, onLoad: ( gltf: GLTF ) => void, onError?: ( event: ErrorEvent ) => void ) : void;
 
 }
+
+export class GLTFParser {
+
+	json: any;
+	extensions: any;
+	options: {
+		path: string,
+		corssOrigin: string,
+		manager: LoadingManager
+	};
```

Comment:
I think I would omit the `extensions` and `options` properties; those aren't really meant to be used externally.

Reply:
sure!